app/src/main/java/com/bleyl/recurrence/database/DatabaseHelper.java

```diff
@@ -119,6 +123,13 @@ public void onUpgrade(SQLiteDatabase database, int oldVersion, int newVersion) {
                     + COL_FRIDAY + " TEXT, "
                     + COL_SATURDAY + " TEXT) ");
         }
+
+        if (oldVersion < 4) {
+            database.execSQL("ALTER TABLE " + NOTIFICATION_TABLE + " ADD " + COL_NAG_TIMER + " INTEGER");
+            database.execSQL("ALTER TABLE " + NOTIFICATION_TABLE + " ADD " + COL_ACTIVE_STATE + " BOOLEAN");
+            database.execSQL("UPDATE " + NOTIFICATION_TABLE + " SET " + COL_NAG_TIMER + " = 0;");
+            database.execSQL("UPDATE " + NOTIFICATION_TABLE + " SET " + COL_ACTIVE_STATE + " = FALSE;");
```

Comment:
There was probably some leftovers of a previous Recurrence installation as this is called only when there's already a database that needs upgrading, and I could reproduce the error when upgrading from the master branch build. Looks like I had forget to put ' in my SQL request, but when I do it triggers another error about an unknown color so there's something wonky going on with that upgrade procedure.

I had no problem installing without any previous version leftovers.